enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/util.tsx

```diff
@@ -0,0 +1,80 @@
+import type { ReactNode } from "react";
```

Comment:
This file here looks a bit mis-named/mis-placed but I can see why it's hard to give it a name/place.

I have an alternative: [add sdk theme selector to storybook](https://github.com/metabase/metabase/pull/51122)